### PR TITLE
Upgrade APL Suggester and Web Viewhost to support APL 1.7

### PR DESCRIPTION
**Pull request by VennSun**
## Description

Upgrade APL from 1.6 to 1.7 with APL suggester and APL Web Viewhost

## Testing

APL E2E sanity tests performed.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly
- [x] I have read the **README** document
- [ ] I have added tests to cover my changes
- [x] All new and existing tests passed
- [x] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

## License

- [x] By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,8 +1,15495 @@
 {
     "name": "alexa-skills-kit-toolkit",
     "version": "2.7.0",
-    "lockfileVersion": 1,
+    "lockfileVersion": 2,
```
Comment (sam-goodwin) [inline package-lock.json:4]: Can we separate concerns here? Let's not conflate a change to the lock file format with releasing APL 1.7.

**Review by sam-goodwin — CHANGES_REQUESTED** [review]

**Comment by sam-goodwin** [comment]
Should there be an update to the CHANGELOG?

**Comment by sam-goodwin** [comment]
> Should there be an update to the CHANGELOG?

Or, I assume, that would be generated when we release. nevermind.

File: package-lock.json (hunk 1 above)
Comment (VennSun) [reply package-lock.json:4]: The package lock file is auto generated. Should I not including the package.lock file in the PR?

Comment (sam-goodwin) [reply package-lock.json:4]: The package lock is because you're using `npm 7`. Try re-installing with `npm 6` for this PR

**Review by sam-goodwin — APPROVED** [review]

**Review by ShenChen-Amazon — APPROVED** [review]

**Comment by ShenChen-Amazon** [comment]
> > Should there be an update to the CHANGELOG?
> 
> Or, I assume, that would be generated when we release. nevermind.

Before release, we need to munually update : https://github.com/alexa/ask-toolkit-for-vscode/blob/development/media/toolkitUpdate/toolkitUpdate.html.

This page is not yet autogenerated

**Pull request merged**